iptables/iptables.go

```diff
@@ -78,7 +79,8 @@ const (
 
 // known IP's
 const (
-	AzureDNS = "168.63.129.16"
+	AzureDNS  = "168.63.129.16"
+	AzureIMDS = "169.254.169.254"
```

Comment:
idk what this is for, specifically, but should we do it for the whole link-local block?

Reply:
This is specifically to allow http traffic to the metadata service from pods. Since NCs are considered "secondary interfaces" and there is an explicit deny rule on the host to drop http traffic from secondary interfaces. So really we only want to snat traffic to this one IP against the host IP , since it gets dropped otherwise. For the rest of link-local there is no explicit deny rule so no need to snat.